### PR TITLE
911 TC-WalkerColina-07

### PR DESCRIPTION
**Pull request by wcolina**
**Test Case:**
image

**Automated Test Case Result:**

image

### CODE REVIEW

File: src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPECreate.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    private WebElement descriptionTextInput;
+
+    @FindBy(id = "cbxPpeClassification")
+    private WebElement ppeClassificationSelectOption;
+
+    @FindBy(id = "btnPpeSave")
+    private WebElement saveButton;
+
+    public PPECreate() {
+        CommonEvents.isPresent(saveButton);
```
Comment (airmiky) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPECreate.java]: usar "isVisible"

File: src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEList.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    private WebElement newPPEButton;
+
+    @FindBy(className = "container")
+    private WebElement ppeContainer;
+
+    @FindBy(xpath = "//table[@id='tblPpe']/descendant::tr[last()]/td[count(//table[@id='tblPpe']/descendant::th[text()='Name'])]")
+    private WebElement lastPPEName;
+
+    public PPEList() {
+        CommonEvents.isPresent(ppeContainer);
```
Comment (airmiky) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEList.java]: usar "isVisible"

File: src/test/resources/features/ppe/PPECreate.feature [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# @Author: Walker Colina
+# feature to review behaviour of create ppe classification
+Feature: PPECreate
+
+  Scenario: La insercion de registro para un de equipo de proteccion personal se muestra en la lista de de equipo de proteccion personal cuando presiono boton Save
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPECreate.feature]: corregir a minuscula "click"

File: src/test/resources/features/ppe/PPECreate.feature [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    And Click 'PPE' sub menu of 'PPE' menu
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPECreate.feature]: corregir a minuscula "click"

File: src/test/resources/features/ppe/PPECreate.feature [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+    And Click 'New PPE' button of 'PPE list' page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPECreate.feature]: corregir a minuscula "click"

File: src/test/resources/features/ppe/PPECreate.feature [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+    And Fill 'PPE' form using Data Driver Test on create 'PPE' page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPECreate.feature]: corregir a minuscula "fill"

File: src/test/resources/features/ppe/PPECreate.feature [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+    And Click 'Save' button into create 'PPE' form page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPECreate.feature]: corregir a minuscula "click"

File: src/test/resources/features/ppe/PPEList.feature [hunk 8]
```diff
@@ -0,0 +1,9 @@
+# @Author: Walker Colina
+# feature to review behaviour of ppe list
+Feature: PPEList
+
+  Scenario: Todos los datos de equipos de protección personal son mostrados en una lista
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEList.feature]: corregir a minuscula "click"

File: src/test/resources/features/ppe/PPEList.feature [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+    And Click 'PPE' sub menu of 'PPE' menu
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEList.feature]: corregir a minuscula "click"

**Review by airmiky — COMMENTED** [review]
Revisar las observaciones por favor.

File: src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPECreate.java (hunk 1 above)
Comment (wcolina) [reply src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPECreate.java]: Done.

File: src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEList.java (hunk 2 above)
Comment (wcolina) [reply src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEList.java]: Done.

File: src/test/resources/features/ppe/PPECreate.feature (hunk 3 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPECreate.feature]: Done.

File: src/test/resources/features/ppe/PPECreate.feature (hunk 4 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPECreate.feature]: Done.

File: src/test/resources/features/ppe/PPECreate.feature (hunk 5 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPECreate.feature]: Done.

File: src/test/resources/features/ppe/PPECreate.feature (hunk 6 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPECreate.feature]: Done.

File: src/test/resources/features/ppe/PPECreate.feature (hunk 7 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPECreate.feature]: Done.

File: src/test/resources/features/ppe/PPEList.feature (hunk 8 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEList.feature]: Done.

File: src/test/resources/features/ppe/PPEList.feature (hunk 9 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEList.feature]: Done.

**Comment by wcolina** [comment]
All comments fixed.

**Pull request merged**